app/models/settings.rb

```diff
@@ -0,0 +1,40 @@
+# == Schema Information
+#
+# Table name: settings
+#
+#  id                  :integer          not null, primary key
+#  created_at          :datetime
+#  updated_at          :datetime
+#  singleton_guard     :integer
+#  current_semester_id :integer
+#  npo_app_open        :boolean
+#  student_app_open    :boolean
+#
+
+class Settings < ActiveRecord::Base
+  validates_numericality_of :singleton_guard, equal: 0
+  validates :npo_app_open, inclusion: { in: [true, false] }
```

Comment:
Correct me if I'm wrong but I think you can just do presence: true instead of the inclusion

http://guides.rubyonrails.org/active_record_validations.html#presence